### ISSUE TITLE
[Support] Change Events To Look At Streamlabs/OBS Instead Of Twitch

### ISSUE THREAD

**Issue by ric-k** [crowbartools/Firebot#1240]
When using Streamlabs to broadcast, some events are not happening in Firebot due to the restriction of Twitch only allowing one request per minute. 'Follow' allows you to select from Streamlabs notification instead as a workaround so multiple events can happen per minute. Can this not be implemented for the other common features like Raid, Subscriber, Host, Bits etc? 

At the moment, I find than if I am raided in Twitch the event I have set will not happen, but when I get a few follows right after the raid, they work fine because they are coming from Streamlabs. Thanks.

**Comment by THECorranHorn** [comment]
Not necessarily on topic but with firebot you can switch a ton of stuff over to firebot that slobs doesn't do very well, and by switching you can simplify your stream system.

**Comment by RickLee84** [comment]
Hi, could you clarify what can be switched over and how this is done? I have went through the Settings menu and didn't see anything obvious for this, but if I can switch more over to Firebot that would make my streaming a lot easier, thanks.

**Comment by ebiggz** [comment]
Hi @RickLee84! Thanks for making an issue. It sounds like you are running into a bug or some other issue as there is not a "one request per minute" restriction for real time events for things such as Raid, Host, Sub, Bits, etc. 

The reason we have a streamlabs counterpart for the Follow event specifically is because the Follow event is the only event that twitch broadcasts in a way that cloud-based tools can easily listen to but desktop-based bots (like firebot) cannot. So we have to poll the API which is slow. So I added the follow counter parts for common third party tools such as Streamlabs to "piggy back" off their realtime access. This is not needed for any other event type though.

**Comment by RickLee84** [comment]
Hi @ebiggz thank you for the reply. Yes it would appear I am encountering a bug or some other issue. The follow notification does appear for me just fine, as do any graphics set for shoutouts or custom commands, but the Raid, Sub, Bits, Host etc. notifications just won't show up. I have followed all the steps given in various YT videos for it and insured that I have connected Streamlabs with Firebot and have 3 green ticks in the bottom left before starting streaming.
